src/reanimated2/UpdateLayoutAnimations.ts

```diff
@@ -0,0 +1,48 @@
+'use strict';
+import { shouldBeUseWeb } from './PlatformChecker';
+import { configureLayoutAnimationBatch } from './core';
+import type {
+  LayoutAnimationFunction,
+  LayoutAnimationType,
+} from './layoutReanimation';
+
+function createUpdateManager() {
+  const animations: {
+    viewTag: number;
+    type: LayoutAnimationType;
+    config?: Keyframe | LayoutAnimationFunction;
+  }[] = [];
+
+  return {
+    update(config: {
+      viewTag: number;
+      type: LayoutAnimationType;
+      config?: Keyframe | LayoutAnimationFunction;
+    }) {
+      animations.push(config);
```

Comment:
Lets do what we talked last time, something like that but with better naming
```suggestion
      config.config = makeShareableCloneRecursive(config.config);
      animations.push(config);
```
and remove `map()` from `core.ts`

Reply:
I changed it a bit differently, I call `makeShareableCloneRecursive` in the `updateLayoutAnimations` function. From there I can always simply pass the batch as it is.